cmd/guacone/cmd/scorecard.go

```diff
@@ -0,0 +1,175 @@
+//
```

Comment:
opening a comment to block kodiakbot cause something changed in default settings and its messing up branch merges

Reply:
it's fixed now.